src/Modules/quorum.ts

```diff
@@ -1,6 +1,42 @@
-import { GraphData } from "../GraphTypes";
+import { GraphData, GraphNode } from "../Types/GraphTypes";
+import { NetworkGraphNode } from "../Types/NetworkTypes";
 import dummydata from "../dummydata";
 
+const networkData = dummydata as { nodes: NetworkGraphNode[] };
+
+function convertResponseToGraphData(json: {
+  nodes: NetworkGraphNode[];
+}): GraphData {
+  const convertedNodes: GraphNode[] = json.nodes.map(gn =>
+    Object.assign({}, gn, { id: gn.node })
+  );
+
+  const data: GraphData = {
+    nodes: convertedNodes,
+    links: []
+  };
+  json.nodes.forEach(n => {
+    let targets: string[] = [];
+    let others = n.qset.v;
+    if (others[0] && others[0] instanceof Object) {
+      (others as { t: number; v: string[] }[]).forEach(other => {
```

Comment:
Should we define a type (maybe `QuorumSet`) for this? It might help simplifying a bit this part of the code.

Reply:
done